src/libraries/System.Console/tests/WindowAndCursorProps.cs

```diff
@@ -102,13 +95,6 @@ public static void WindowHeight_GetUnix_Success()
         Helpers.RunInRedirectedOutput((data) => Console.WriteLine(Console.WindowHeight));
     }
 
-    [Fact]
-    [PlatformSpecific(TestPlatforms.AnyUnix)]  // Expected behavior specific to Unix
-    public static void WindowHeight_SetUnix_ThrowsPlatformNotSupportedException()
```

Comment:
Shouldn't those tests be left here and just changed to only Browser, iOS, MacCatalyst and tvOS?

Reply:
Yes, please.

Reply:
Fixed. Thank you for the info. :)